eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/NDJsonLineMapper.java

```diff
@@ -0,0 +1,15 @@
+package uk.ac.ebi.eva.accession.dbsnp2.io;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.batch.item.file.LineMapper;
+
+public class NDJsonLineMapper implements LineMapper<JsonNode> {
```

Comment:
What does this "ND" mean? Maybe `JsonNodeLineMapper` would be more obvious.

Reply:
ND means New-line Delimited, which is the format for the DbSNP JSON files - renamed to `JsonNodeLineMapper`